R/vec-utils.R

```diff
@@ -118,6 +128,16 @@ seq2_along <- function(from, x) {
   seq2(from, length(x))
 }
 
+# Better argument ordering and empty punning when no match
+match2 <- function(vec, elt) {
```

Comment:
Would this be better as `match1()` since I think you're implying the `elt` is length 1?

Reply:
Good point I hadn't considered the vector elements case.